tools/Utils/c_rehash.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/python
+
+import os
+import argparse
+import subprocess
+
+def process_file (dirname, file) :
+    # Old full file ocation
+    full_path = dirname + "\\" + file
```

Comment:
@dev-gh , no , one \ is service symbol, and its screens next one. in construction "\" its syntax error.